make.go

```diff
@@ -489,7 +482,7 @@ func writeTemplates(dir, gopkg, debsrc, debbin, debversion string, dependencies
 	builddeps := append([]string{"debhelper (>= 10)", "dh-golang", "golang-any"}, dependencies...)
 	fmt.Fprintf(f, "Build-Depends: %s\n", strings.Join(builddeps, ",\n               "))
 	fmt.Fprintf(f, "Standards-Version: 4.0.0\n")
-	fmt.Fprintf(f, "Homepage: %s\n", websiteForGopkg(gopkg))
+	fmt.Fprintf(f, "Homepage: %s\n", getHomepageForGopkg(gopkg))
```

Comment:
Why did you rename this function? Can we keep the old name to make this change minimal?